coalib/settings/ConfigurationGathering.py

```diff
@@ -122,6 +124,16 @@ def warn_config_absent(sections, argument, log_printer):
                          "to give the `--{}` argument?".format(argument))
 
 
+def configure_logging_from_json(filename):
+    """
+    Configures logging module based on parameters from JSON file.
+
+    :param filename:    Name of a file with logging configuration.
+    """
+    with open(filename) as cnf_file:
+        logging.config.dictConfig(json.load(cnf_file))
```

Comment:
The separation of concerns requirement can be satisfied by just moving this to its own module. That would still save us the trouble of
- Managing files in the code manually
- Having to bring in JSON serialization
- Having to write JS code

But anyway, I don't think 'separation of concerns' applies at the level. Loading configuration (`ConfigurationGathering.py`) and logging configuration (`logging_config.json`) are the same concern, so it might be okay. The issue would be if the config gatherer suddenly started installing bears that are mentioned in the config file or something :)